netlify.toml

```diff
@@ -3,6 +3,9 @@
   publish   = "site/dist"
   command   = "gulp build"
 
+[build.environment]
```

Comment:
Are you sure this doesn't happen by default?

Reply:
It does, but we need to move devDeps -> deps

Reply:
From https://www.netlify.com/docs/build-settings/:
> `NODE_ENV` is by default set to `development` (typical settings: production or development; setting globally allows all dependency installations to work with our caching.)

You are right, running `npm install` with `NODE_ENV` set to `production` should ignore dev dependencies. We can either use another var name, set it only for the build command (in the `command` field?) we move the devdeps to deps.

Reply:
Another option is to enable cachebusting within the file, although I think `npm run build` would be better

When running `gulp watch`, set the `SFD_CACHEBUST_DISABLED` to true and check for it